x-pack/plugins/cases/common/utils/owner.ts

```diff
@@ -6,9 +6,24 @@
  */
 
 import { OWNER_INFO } from '../constants';
+import type { Owner } from '../constants/types';
 
 export const isValidOwner = (owner: string): owner is keyof typeof OWNER_INFO =>
   Object.keys(OWNER_INFO).includes(owner);
 
 export const getCaseOwnerByAppId = (currentAppId?: string) =>
   Object.values(OWNER_INFO).find((info) => info.appId === currentAppId)?.id;
+
+export const getOwnerFromRuleConsumerProducer = (consumer?: string, producer?: string): Owner => {
+  for (const value of Object.values(OWNER_INFO)) {
+    const foundedConsumer = value.validRuleConsumers?.find(
```

Comment:
I tested with `oblt` serverless and created a case template. Then was able to see that template in case action. But I have a feeling that your bug scenario was little bit different. Don't remember the exact scenario.

Reply:
but could you see the case that was created when the rule triggered?

Reply:
yup, case was created.

Reply:
i think that was the case, ill double check when you merge this!